src/schema/firefox-schemas-import.js

```diff
@@ -26,6 +26,9 @@ export const refMap = {
 // Reference some functions on inner so they can be stubbed in tests.
 export const inner = {};
 
+// Consider moving this to a Set if you add more schema namespaces.
+export const ignoredSchemas = ['omnibox_internal'];
```

Comment:
This schema namespace isn't actually exposed to add-ons, or they shouldn't be using it if it is. I'll update the comment.